packages/Makefile

```diff
@@ -1,6 +1,7 @@
 export PYODIDE_ROOT=$(abspath ..)
 PYODIDE_LIBRARIES=$(abspath ./.artifacts)
 CPYTHONROOT=$(PYODIDE_ROOT)/cpython
+HOSTPYTHON=$${HOSTPYTHON:-python3.9}
```

Comment:
It's already defined in Makefile.envs I believe

https://github.com/pyodide/pyodide/blob/b205f8e6bd176b02e701407099843aa8900fe123/Makefile.envs#L19

Why would we need to re-define it here, particularly before including the Makefile.envs ?